tests.py

```diff
@@ -37,7 +37,7 @@ def test_insert_delete():
         TestHelper.test_root(tree, 11)
 
         # test deletions
-        TestHelper.test_deletion(tree, 1, 0)
+        TestHelper.test_deletion(tree, 1, 2)
```

Comment:
The documentation says `delete` should return number of rebalancing actions. When 1 is removed in this case, we simply delete the node without any rebalancing actions. Shouldn't this be 0?

Reply:
This is correct, my bad